README.md

```diff
@@ -166,6 +166,33 @@ yarn run js:lint
 yarn run sass:lint
 ```
 
+### RSpec parallel testing
+The service include [parallel_tests](https://github.com/grosser/parallel_tests) gem, that allows to split the RSpec test
+suite run accross multiple CPU cores.
+
+Each group will run against a separate database.
+To set it up:
+
+```
+RAILS_ENV=test rails parallel:create
```

Comment:
RAILS_ENV=test isn't actually needed (but it doesn't hurt)

Reply:
I had issues with `rails/rake parallel` setup commands running under `development` by default. Maybe something about the devcontainers though :shrug: